oe_translation.install

```diff
@@ -145,3 +145,73 @@ function oe_translation_update_9001(): void {
   $update_manager->installEntityType($translation_request_log);
 
 }
+
+/**
+ * Add index to all fields of type oe_translation_entity_revision_type_item.
+ */
+function oe_translation_update_9002(): void {
+  /** @var \Drupal\Core\Entity\EntityFieldManager $field_manager */
+  $field_manager = \Drupal::service('entity_field.manager');
+  $definition_manager = \Drupal::entityDefinitionUpdateManager();
+  $database = \Drupal::database();
+  $entity_type_manager = Drupal::entityTypeManager();
+  $field_type = 'oe_translation_language_with_entity_revision';
+  // Get all field storage configs for this field type.
+  $field_storage_configs = \Drupal::entityTypeManager()
+    ->getStorage('field_storage_config')
+    ->loadByProperties(['type' => $field_type]);
+
+  foreach ($field_storage_configs as $field_storage) {
+    $field_name = $field_storage->getName();
+    $entity_type_id = $field_storage->getTargetEntityTypeId();
+    $storage = $entity_type_manager->getStorage($entity_type_id);
+    $table_mapping = $storage->getTableMapping();
+    $table_name = $table_mapping->getFieldTableName($field_name);
+    $table_schema = $field_storage->getSchema();
+    $spec_properties = ['entity_type', 'entity_id'];
+    $index_columns = [
+      'field_language_revision_entity_type',
+      'field_language_revision_entity_id',
+    ];
+
+    $spec = [];
+    for ($i = 0; $i < count($index_columns); $i++) {
```

Comment:
Minor, but count() gets called during every single cycle. A variable for this would make more sense.